test/unit/csi-daemonset.bats

```diff
@@ -535,6 +535,28 @@ load _helpers
   [ "${actual}" = "true" ]
 }
 
+#--------------------------------------------------------------------
+# hostNetwork
+
+@test "csi/daemonset: csi.hostNetwork not set" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      --show-only templates/csi-daemonset.yaml \
+      . | tee /dev/stderr |
```

Comment:
I think these will need csi.enabled=true set since by default the csi daemonset is not deployed in the chart.

```suggestion
  local actual=$(helm template \
      --show-only templates/csi-daemonset.yaml \
      --set 'csi.enabled=true' \
      . | tee /dev/stderr |
```

Reply:
done